packages/web-app-files/src/components/SideBar/Shares/NewCollaborator/Form.vue

```diff
@@ -86,15 +71,17 @@ import { roleToBitmask } from '../../../../helpers/collaborators'
 import { shareTypes } from '../../../../helpers/shareTypes'
 
 import AutocompleteItem from './AutocompleteItem.vue'
-import CollaboratorsEditOptions from './CollaboratorsEditOptions.vue'
+import RoleSelection from '../RoleSelection.vue'
 import RecipientContainer from './RecipientContainer.vue'
+import ExpirationDatepicker from './ExpirationDatepicker.vue'
 
 export default {
-  name: 'NewCollaborator',
+  name: 'NewCollaboratorForm',
```

Comment:
I like NewCollaboratorForm, but should we name the file accordingly?

Reply:
Sure, I can take care of that! Any other suggestions regarding naming? I'm not 100% happy with what I've got so far

Reply:
Maybe AddNewCollaboratorForm?

Reply:
... or `InviteCollaboratorForm`?

Reply:
yes that sounds good, I would also rename the folder from NewCollaborator to InviteCollaborator sounds more intuitive